### ISSUE TITLE
fail run on local

### ISSUE THREAD

**Issue by satrioaw** [sveri/closp-docs#1]
I clone on my local then execute command 

$lein run

found this message

```
tempe@goreng:~/c/closp-docs:master$ lein run

loading module: Markdown parser

compiling assets...

copying theme resources

copying resources

Error: resource resources/templates/md/posts not found

"Elapsed time: 339.970441 msecs"
```

**Comment by sveri** [comment]
Hi,

As the error message says, it misses the folder:  resources/templates/md/posts
I think you will have to create it locally.

Also I guess the correct command to compile everything is by running: lein ring server

**Comment by satrioaw** [comment]
working now thanks, any idea for multiple language translation ? just copy paste then translate ?

**Comment by sveri** [comment]
Good to hear.

I never thought about translation. I would suggest you play around with it a bit and see if you can come up with a viable solution.
My first thought would be to copy the pages and add "-language-identifier" like "-de". Later I could parse the pages and build a different server structure on top of it. 

Also, if you want to take the extra mile, go here: https://github.com/cryogen-project/cryogen and ask if it somehow supports multiple languages out of the box.Cryogen is what the docs are based on.

Maybe it would also be possible to do some regex replacement on server side.

Tldr; I have no final solution, we need to figure out how to do it.
